### DISCUSSION TITLE
YouTube/Mixcloud Plugin not working

### DISCUSSION THREAD

**Discussion by mothabrain** [LMS-Community/slimserver#1271] [General]
Hi there! 

Anyone also experiencing playback problems with the yt and/or mixcloud plugin? 

For me it started today on "Version: 9.1.0 - 1735392763 @ Sat Dec 28 13:33:57 UTC 2024". 
On older versions it doesn't work as well.

I even cannot provide any logging hints so i appreciate any tips for the problem.

**Comment by michaelherger** [comment]
Hi there - I'd recommend you join forums.lyrion.org. You'll find more helpful people there.
